adhocteam/Head-Start-TTADP
# Assign regions and grantees to cdi grants

**Description of change**

The Admin UI allows CDI grants to be assigned a region and grantee. When a CDI grant is encountered (a grant in region 13) while importing a CDI flag is set to true (and the region/grantee are not updated). The admin UI looks for this flag to allow the grant to be managed. After being assigned a grantee and region the grant can be selected in an activity report.

**How to test**

1) Pull down changes
2) Run migrations and re-seed your database
3) Go to the admin page and assign regions/grantees to CDI grants
4) Verify you can select a CDI grant in a report (you'll have to assign that grant to region 1 to have it select-able in the report, assuming your user is also assigned region 1)

or play around on [sandbox](https://tta-smarthub-sandbox.app.cloud.gov/)

**Issue(s)**
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-93

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
